db/fixtures/ae_datastore/ManageIQ/Infrastructure/VM/Provisioning/Placement.class/__methods__/vmware_best_fit_least_utilized.rb

```diff
@@ -6,7 +6,7 @@ def root_folder(host, data_center)
 end
 
 def set_folder(prov, host, vm)
-  host_dc = host.ems_cluster.v_parent_datacenter
+  host_dc = host.v_owning_datacenter.presence || host.ems_cluster.v_parent_datacenter
```

Comment:
@Fryguy I worked on this yesterday with @billfitzgerald0120.  I will remove the workaround and fix the underlying method in a followup PR.